Makefile

```diff
@@ -145,7 +145,7 @@ test-fullstack:
 
 .PHONY: test-scheduler
 test-scheduler:
-	$(MAKE) test-with-database SCHEDULER_FULLSTACK=1 PROVE_ARGS="$$HARNESS t/05-scheduler-full.t" RETRY=3
+	$(MAKE) test-with-database SCHEDULER_FULLSTACK=1 SCALABILITY_FULLSTACK_WORKER_COUNT=5 SCALABILITY_FULLSTACK_JOB_COUNT=10 PROVE_ARGS="$$HARNESS t/05-scheduler-full.t t/43-scheduling-and-worker-scalability.t" RETRY=3
```

Comment:
I think it's a good idea to schedule this as part of the other "scheduler" test to better share load among the different test jobs. However, would you mind putting the configuration values into the test itself as "good default values"? I don't think it should make a difference for someone if that person calls "prove" manually or "make test-scheduler". After all, it's still just default values and they can be easily overriden with environment variables.